core/src/avm1/globals/gradient_bevel_filter.rs

```diff
@@ -347,7 +354,7 @@ pub fn set_strength<'gc>(
         .get(0)
         .unwrap_or(&1.into())
         .coerce_to_f64(activation)
-        .map(|x| x.clamp(0.0, 255.0))?;
+        .map(|x| x.max(0.0).min(255.0))?;
```

Comment:
why is the clamp change needed?

Reply:
If x is `f64::NAN`, using `.clamp()` returns NaN ; using `.max(0.0).min()` returns 0. In Flash Player, this property returns 0 in this situation. Properties like `blurX` and `blurY` return NaN, so they remain unchanged and still use `.clamp()`.

Reply:
How about putting it in a separate function, with a comment repeating what you wrote here? Either freestanding `nan_lossy_clamp(x, 0.0, 255.0)` or via trait `x.nan_lossy_clamp(0.0, 255.0)`. Name subject to change ;D

This also lets you have only a single clippy allow attribute instead of having it everywhere.